app/src/main/java/de/danoeh/antennapod/adapter/DownloadStatisticsListAdapter.java

```diff
@@ -40,7 +41,10 @@ String getHeaderValue() {
 
     @Override
     void onBindFeedViewHolder(StatisticsHolder holder, StatisticsItem item) {
-        holder.value.setText(Formatter.formatShortFileSize(context, item.totalDownloadSize));
+        holder.value.setText(Formatter.formatShortFileSize(context, item.totalDownloadSize)
+                + "/"
```

Comment:
Could you please replace this with the middle dot and space characters, eg. ` · `?